peerdiversity/filter.go

```diff
@@ -220,6 +220,13 @@ func (f *Filter) ipGroupKey(ip net.IP) (PeerIPGroupKey, error) {
 		if err != nil {
 			return "", fmt.Errorf("failed to fetch ASN for IPv6 addr %s: %w", ip.String(), err)
 		}
+
+		// if no ASN found then fallback on using the /32
+		if len(s) == 0 {
+			dfLog.Debugw("group key is empty", "appKey", f.logKey, "ip", ip.String())
```

Comment:
There's no peerID in the error here like the equivalent error used to have, but this is likely more about the ASN table being out of date than the peerID. Thoughts?

Reply:
Eh, I don't think the peer ID is really all that useful anyways. All we care about is the IP.